### PR TITLE
chore(tabs, in page nav): update styles

### PR DESCRIPTION
**Pull request by nkrantz**
In Page Nav
- Update styles to match Figma (vertical and horizontal)
- Update docs

Tabs
- Update styles to match Figma (vertical and horizontal)
- Add `full_width` and `inverse_full_width` variants (same as fitted)
- Update docs

Tokens
- Add 3 new box shadow tokens for the top borders of horizontal selected and selected/hover tabs

Box
- Add 4 new pseudo style selectors for adding the nubbin to vertical tabs

### CODE REVIEW

File: packages/paste-core/components/in-page-navigation/src/InPageNavigationItem.tsx [hunk 1]
```diff
@@ -96,6 +123,13 @@ export interface InPageNavigationItemProps extends HTMLPasteProps<"a"> {
    * @memberof InPageNavigationItemProps
    */
   href: string;
+  /**
+   * Accessible title for when the InPageNavigationItem is truncated. Usage is strongly recommended on all InPageNavigationItems, but especially when the text might be truncated (in vertical InPageNavigations or horizontal ones with more than 3 items).
```
Comment (SiTaggart) [inline packages/paste-core/components/in-page-navigation/src/InPageNavigationItem.tsx:127]: praise: loving our ability to write better api docs

File: .changeset/two-numbers-shake.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@twilio-paste/box": minor
+"@twilio-paste/core": minor
+---
+
+[Box] Add four new pseudo prop style selectors for use in the tabs package.
```
Comment (SiTaggart) [inline .changeset/two-numbers-shake.md:6]: suggestion: tell me what they are so I know to look for them

File: .eslintrc.js [hunk 3]
```diff
@@ -68,6 +68,8 @@ module.exports = {
           "_selected_hover",
           "_selected_focus",
           "_selected_focusVisible",
+          "_selected_after",
+          "_focus_selected_after",
```
Comment (SiTaggart) [inline .eslintrc.js:72]: observation: weird that you added four, but eslint only cared about two of them

Comment (nkrantz) [reply .eslintrc.js:72]: yeah that's why the error was throwing me off

File: .changeset/two-numbers-shake.md (hunk 2 above)
Comment (nkrantz) [reply .changeset/two-numbers-shake.md:6]: done

**Review by SiTaggart — APPROVED** [review]

File: packages/paste-core/components/tabs/src/Tab.tsx [hunk 4]
```diff
@@ -37,21 +38,47 @@ const getTabBoxStyles = (orientation: Orientation, variant: Variants): BoxStyleP
...
-          borderRadius: "borderRadius20",
+          borderTopLeftRadius: "borderRadius30",
+          borderBottomLeftRadius: "borderRadius30",
+        },
+        _selected_after: {
+          content: '" "',
+          position: "absolute",
+          height: "80%",
+          width: "4px",
+          backgroundColor: isInverse ? "colorBackgroundPrimaryWeakest" : "colorBackgroundPrimary",
```
Comment (nkrantz) [inline packages/paste-core/components/tabs/src/Tab.tsx:61]: change colorBackgroundPrimaryWeakest to use colorBackgroundInverseStrongest (new token, same value, gray 0)

File: packages/paste-core/components/tabs/src/Tab.tsx [hunk 5]
```diff
@@ -85,13 +113,13 @@ const getTabBoxStyles = (orientation: Orientation, variant: Variants): BoxStyleP
...
         _hover: {
-          color: isInverse ? "colorTextInverse" : "colorTextPrimary",
-          borderBottomColor: isInverse ? "colorBorderInverseStrongest" : "colorBorderPrimary",
+          color: isInverse ? "colorTextInverse" : "colorTextPrimaryStrongest",
         },
         _selected_hover: {
           borderColor: isInverse ? "colorBorderInverseStrongest" : "colorBorderWeak",
           color: isInverse ? "colorTextInverse" : "colorTextPrimaryStronger",
           borderBottomColor: isInverse ? "colorBorderInverseWeakest" : "colorBorderWeakest",
+          boxShadow: isInverse ? "shadowBorderTopPrimary" : "shadowBorderTopPrimaryStronger",
```
Comment (nkrantz) [inline packages/paste-core/components/tabs/src/Tab.tsx:122]: - rename shadowBorderTopPrimaryStronger to shadowBorderTopPrimaryStrongest
- rename shadowBorderInverseTopPrimary to shadowBorderTopInverseStrongest

File: packages/paste-website/src/pages/components/tabs/index.mdx [hunk 6]
```diff
@@ -141,9 +141,9 @@ Use Horizontal Tabs in most cases, as they most realistically mimic a physical t
   {HorizontalTabsExample}
 </LivePreview>
 
-### Fitted Tabs
+### Full width Tabs
```
Comment (serifluous) [inline packages/paste-website/src/pages/components/tabs/index.mdx:144]:
```suggestion
### Full-width Tabs
```

File: packages/paste-website/src/pages/components/tabs/index.mdx [hunk 7]
```diff
@@ -141,9 +141,9 @@ Use Horizontal Tabs in most cases, as they most realistically mimic a physical t
...
 
-Use Fitted Tabs to fill a parent element with a set width. Each Tab will equally expand and fit the width of the containing element. Fitted Tabs are recommended for compact spaces.
+Use full width Tabs to fill a parent element with a set width. Each Tab will equally expand and fit the width of the containing element. Full width Tabs are recommended for compact spaces.
```
Comment (serifluous) [inline packages/paste-website/src/pages/components/tabs/index.mdx:146]:
```suggestion
Use full-width Tabs to fill a parent element with a set width. Each Tab will equally expand and fit the width of the containing element. Full-width Tabs are recommended for compact spaces.
```

File: packages/paste-core/components/tabs/src/Tab.tsx [hunk 8]
```diff
@@ -37,21 +38,47 @@ const getTabBoxStyles = (orientation: Orientation, variant: Variants): BoxStyleP
...
+          right: -1,
+        },
+        _focus_selected_after: {
+          display: "none",
+        },
+        _focus_selected_hover_after: {
+          display: "none",
         },
       };
     case "horizontal":
```
Comment (serifluous) [inline packages/paste-core/components/tabs/src/Tab.tsx:84]: chore: change inverse text color (default state) to `colorTextInverseWeaker`

Forgot to spec this out.

File: packages/paste-core/components/tabs/src/Tab.tsx [hunk 9]
```diff
@@ -85,13 +113,13 @@ const getTabBoxStyles = (orientation: Orientation, variant: Variants): BoxStyleP
         },
         /* Hover states */
         _hover: {
-          color: isInverse ? "colorTextInverse" : "colorTextPrimary",
-          borderBottomColor: isInverse ? "colorBorderInverseStrongest" : "colorBorderPrimary",
+          color: isInverse ? "colorTextInverse" : "colorTextPrimaryStrongest",
         },
         _selected_hover: {
           borderColor: isInverse ? "colorBorderInverseStrongest" : "colorBorderWeak",
```
Comment (serifluous) [inline packages/paste-core/components/tabs/src/Tab.tsx:119]: chore: This border color change can be removed.

**Review by SiTaggart — DISMISSED** [review]

File: packages/paste-core/components/tabs/src/Tab.tsx (hunk 8 above)
Comment (serifluous) [reply packages/paste-core/components/tabs/src/Tab.tsx:84]: nvm `colorTextInverseWeak`

**Review by SiTaggart — APPROVED** [review]

**Review by serifluous — APPROVED** [review]

**Pull request merged**